package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screwdriver-models",
-  "version": "10.4.0",
+  "version": "11.0.0",
```

Comment:
Because it needs a new executor version passed to it.  I can't declare that as a peer dep.